Makefile

```
@@ -1,14 +1,23 @@
+CONTAINER_TOOL := docker
+
+podman: CONTAINER_TOOL = podman
+podman: build-test check-podman
+
 all: build-test check
```

```suggestion
all: build-test check

podman: CONTAINER_TOOL = podman
podman: build-test check-podman
```
Oh, I've overlooked this. You should swap these targets, otherwise, plain `make` executes the first target it sees.